app/src/main/java/org/p2p/wallet/auth/model/DeviceShareKey.kt

```diff
@@ -6,6 +6,7 @@ data class DeviceShareKey(
     @SerializedName("reconstructedETH") val reconstructedKey: String,
     @SerializedName("privateSOL") val privateKey: String,
     @SerializedName("deviceShare") val share: DeviceShare?,
+    @SerializedName("userId") var userId: String?,
 ) {
     data class DeviceShare(
```

Comment:
i would really want to special name entities that we serialize in shared prefs, what do you think?

Reply:
what names ?
I do not get it